inference-engine/src/low_precision_transformations/src/transformer.cpp

```diff
@@ -327,6 +329,7 @@ TypeRelaxedReplacer::TypeRelaxedReplacer() {
     make_matcher_type_relaxed<opset1::Clamp>(this);
     make_matcher_type_relaxed<opset1::Concat>(this);
     make_matcher_type_relaxed<opset1::Convolution>(this);
+    make_matcher_type_relaxed<opset1::ConvolutionBackpropData>(this);
```

Comment:
Use `NGRAPH_RTTI_DECLARATION` for runtime type info declaration of type relaxed `Deconvolution` operation, please. See `Convolution` as example.

Reply:
Did you mean for `ConvolutionBackpropData`? It has been done already, I suppose.

Reply:
I was wrong, was discussed offline - it's not an issue, thanks!